config/vufind/RateLimiter.yaml

```diff
@@ -134,3 +149,26 @@ Policies:
       policy: token_bucket
       limit: 2000
       rate: { interval: '10 minutes', amount: 400 }
+
+# Cloudflare Turnstile is a cloud-based "CAPTCHA alternative" to detect bots.
+# https://developers.cloudflare.com/turnstile/
+# See also [Captcha] settings in config.ini for actual CAPTCHAs.
+# These are the global settings for Turnstile. See also the turnstile config above:
+# - within each policy, to enable turnstile for that policy
+# - storage settings for the result cache
+# And see the required config in contentsecuritypolicy.ini if CSP is used.
+# Turnstile:
```

Comment:
Is this one last key that should be uncommented? Does it possibly make sense to have this uncommented by default so the empty section is in place, so users don't get confused if they uncomment settings within the section and see no results, or would doing that have negative side effects?

Reply:
I removed the space before Turnstile.  I think better to leave it commented out for consistency with the settings below, but if that's not our normal practice I can certainly go the other way.

Reply:
That's fine with me; thanks!